package-lock.json

```diff
@@ -6032,7 +6032,7 @@
 		},
 		"browserify-aes": {
 			"version": "1.2.0",
-			"resolved": "https://registry.npmjs.org/browserify-aes/-/browserify-aes-1.2.0.tgz",
+			"resolved": "http://registry.npmjs.org/browserify-aes/-/browserify-aes-1.2.0.tgz",
```

Comment:
Undo?

Reply:
Sure, this one is very annoying, I observe those differences quite often :)